test/data/devicegraphs/output/s390_dasd_zipl-lvm-sep-home.yml

```diff
@@ -0,0 +1,43 @@
+---
+- dasd:
```

Comment:
NP: I can live with these files as they are. But they look too much like a machine dumb. Removing accessory information like the `partition_table` line, the `start` positions, the redundant partition types and so on would be welcome. :wink: